server/google.go

```diff
@@ -122,7 +122,7 @@ func GetLoginURL(state string) string {
 	return conf.AuthCodeURL(state)
 }
 
-func Auth() gin.HandlerFunc {
+func Auth() func(ctx *gin.Context) {
```

Comment:
Why do we need to change this?

Reply:
in [line 134](https://github.com/rtfpessoa/roodle/pull/5/files#diff-5efd4fd9208dae00a924b10a8f6f9807c78e5522dbe087646a42ef726c493f5eR134), we also needed a `func(c *gin.Context)` and wasn't able to resolve `gin.HandlerFunc` to `type MiddlewareFunc func(c *gin.Context)`